pkgs/development/libraries/kde-frameworks/fetch.sh

```diff
@@ -1 +1 @@
-WGET_ARGS=( https://download.kde.org/stable/frameworks/5.76/ -A '*.tar.xz' )
+WGET_ARGS=( https://download.kde.org/stable/frameworks/5.78/ -A '*.tar.xz' )
```

Comment:
Frameworks 5.79.0 was [just released](https://kde.org/announcements/frameworks/5/5.79.0/) and for kdnssd and syntax-highlighting a 5.79.1 release did follow immediately with hotfixes.